src/vector/text/vtextshaper.cpp

```diff
@@ -0,0 +1,278 @@
+#include "vtextshaper.h"
```

Comment:
you can put this implementation under the flag #ifdef LOTTIE_FONTPATH_SUPPORT , otherwise will be build break 
when we build with meson -Dext-font=false build.

maybe for the next patch :)